sdk/eventhub/azure-eventhub/azure/eventhub/_producer_client.py

```diff
@@ -366,8 +363,11 @@ def _start_producer(self, partition_id, send_timeout):
                     send_timeout=send_timeout,
                 )
 
-    def _create_producer(self, partition_id=None, send_timeout=None):
-        # type: (Optional[str], Optional[Union[int, float]]) -> EventHubProducer
+    def _create_producer(
+            self,
+            partition_id: Optional[str]=None,
+            send_timeout: Optional[float]=None
```

Comment:
nit: sorry I really like spaces

```suggestion
            partition_id: Optional[str] = None,
            send_timeout: Optional[float] = None
```